packages/shared/lib/core/wallet/utils/getActivityTileTitle.ts

```diff
@@ -36,6 +36,11 @@ export function getActivityTileTitle(activity: Activity): string {
             return isConfirmed ? 'general.accountCreated' : 'general.creatingAnAccount'
         }
         return isConfirmed ? 'general.minted' : 'general.minting'
+    } else if (type === ActivityType.Nft) {
```

Comment:
Don't you maybe want to keep the old logic? https://github.com/iotaledger/firefly/blob/c3089e720c571b3c24b319a5d8614b5fe8367d48/packages/shared/lib/core/wallet/utils/getActivityTileTitle.ts#L30

Both look good to me anyway

Reply:
I have followed what we had done with `ActivityType.Account` so that i can use the customized text of `NFT created` instead of simply `mint`... also the line you mentioned no longer exists in `develop-2.0` :see_no_evil: :pray: